frontends/web/src/routes/exchange/components/sell.tsx

```diff
@@ -0,0 +1,211 @@
+import { useNavigate } from 'react-router-dom';
+import { useLoad } from '../../../hooks/api';
+import * as exchangesAPI from '../../../api/exchanges';
+import { findBestDeal, findLowestFee, getSellExchangeSupportedAccounts } from '../utils';
+import { useContext, useEffect, useState } from 'react';
+import { FrontendExchangeDealsList } from '../types';
+import { ExchangeSelectionRadio } from './exchangeselectionradio';
+import { Button } from '../../../components/forms/button';
+import { useTranslation } from 'react-i18next';
+import { IAccount } from '../../../api/account';
+import { getVersion } from '../../../api/bitbox02';
+import style from '../exchange.module.css';
+import { AppContext } from '../../../contexts/AppContext';
+
+type TProps = {
+  accounts: IAccount[];
+  accountCode: string;
+  deviceIDs: string[];
+  selectedRegion: string;
+  onSelectExchange: (exchange: string) => void;
+  selectedExchange: string;
+}
+
+type TExchangeNotAvailableReason = 'coinCode' | 'region' | 'version';
+
+export const Sell = ({
+  accounts,
+  accountCode,
+  deviceIDs,
+  selectedRegion,
+  onSelectExchange,
+  selectedExchange
+}: TProps) => {
+  const { t } = useTranslation();
+  const { setFirmwareUpdateDialogOpen } = useContext(AppContext);
+
+  const exchangeDeals = useLoad(() => exchangesAPI.getExchangeDeals());
```

Comment:
exchange deals endpoint is now only about buy deals. How do we want to handle this? 

Option 1: We could split into two endpoints getExchangeBuyDeals and getExchangeSellDeals
Option 2: we could add a param field for the type of deal (buy/sell) we are looking for
Option 2: We could add a "type" field to the Deal struct (this looks more cumbersome to me).

I like option 2, wdyt?

Reply:
Yes, let's do the second option.